catalog-info.yaml

```
@@ -0,0 +1,28 @@
+---
+ # yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+ apiVersion: backstage.io/v1alpha1
+ kind: Resource
+ metadata:
+   name: buildkite-pipeline-package-registry
+   description: Buildkite Pipeline for package-registry
```

nit. Just update based on the jhttps://github.com/elastic/package-registry/blob/main/.ci/jobs/package-registry.yml#L5enkins definition

```suggestion
   description: Buildkite Pipeline for the Package Registry project
```